SQL/paradise_schema.sql

```diff
@@ -79,6 +79,7 @@ CREATE TABLE `characters` (
   `hair_gradient_offset` varchar(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0,0',
   `hair_gradient_colour` varchar(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '#000000',
   `hair_gradient_alpha` tinyint(3) UNSIGNED NOT NULL DEFAULT '255',
+  `runechat_color` varchar(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '#000000',
```

Comment:
Default this to something other than black *please* otherwise no one will have readable runechat. Make it `#fffff` at a minimum.

Reply:
Defaulted to #FFFFFF, white for readability.